src/routes/choice-exercises/components/SingleChoiceExercise.tsx

```diff
@@ -118,14 +140,14 @@ const ChoiceExerciseScreen = ({
         delayPassed={delayPassed}
       />
       <ButtonContainer>
-        {selectedAnswer !== null && (
+        {selectedAnswer !== null ? (
           <Button onPress={onFinishWord} buttonTheme={BUTTONS_THEME.dark}>
             <>
```

Comment:
Empty Tags are not needed here, are they?

Reply:
They are since we only expect a React Element
```js
interface IButtonProps {
  onPress: () => void
  disabled?: boolean
  children: ReactElement
  buttonTheme?: ButtonThemeType
  testID?: string
}
```
I mean it would make sense to change the interface to `ReactNode` and remove fragment